public/todo.html

```diff
@@ -5,25 +5,43 @@
     <title>TODOS</title>
 </head>
 <body>
-    <main class="container">
-        <h1 class="title">My ToDos</h1>
+    <header>
+        <h1 class="title">Things to do:</h1>
+    </header>
+    <main>
         <section class="todo-list">
             <ul>
                 <li><a href="/todo/1" rel="item"> Go to target</a></li>
+                <li><a href="/todo/2" rel="item"> Read a book</a></li>
+                <li><a href="/todo/3" rel="item"> Walk the dog</a></li>
             </ul>
         </section>
+        <section>
```

Comment:
This might be a good place to use `nav`?  Not a big deal; it might just fit in with the notion of other things related to semantic HTML.